Formula/notcurses.rb

```diff
@@ -30,6 +30,11 @@ def install
     system "cmake", "--install", "build"
   end
 
+  patch do
+    url "https://nick-black.com/strndup-mingw-only.patch"
+    sha256 "8ea4a3be2181e1091e44868646830ea37f2efcfcde984a57e5d8dd48d6bb43e0"
+  end
+
```

Comment:
You can move this just before `def install`.

Reply:
done, thanks